src/commands/new.ts

```diff
@@ -165,5 +168,37 @@ export class NewCommand extends Command {
       outputFile: basename(config.testPath),
       outputDir: dirname(config.stackPath),
     });
+
+    if (config.init) {
+      cli.action.start(
+        chalk.yellow("Installing dependencies. This may take a while...")
+      );
+      await execute("./script/setup", [], { cwd: config.cdkDir });
+      cli.action.stop();
+    }
+
+    // Run `eslint --fix` on the generated files instead of trying to generate code that completely satisfies the linter
+    await execute(
+      "./node_modules/.bin/eslint",
+      [
+        "lib/** bin/**",
+        "--ext .ts",
+        "--no-error-on-unmatched-pattern",
+        "--fix",
+      ],
+      {
+        cwd: config.cdkDir,
+      }
+    );
```

Comment:
There might be an argument for us to produce a `script/fix` command to run eslint and prettier fix?

Reply:
I like that idea. Alternative is to support flags on the scripts, but that is less discoverable - as would require either docs or people to read the script directly.